scripts/node/refreshDatabase.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+const Script = require('./Script');
+
+class RefreshDatabaseScript extends Script {
+  config = {
+    command: '* <dumpPath>',
+    options: {
+      migrate: {
```

Comment:
By the way this option doesn't work on Windows, because we use UNIX paths in the `migrate` command (`'../../babel.config.json'). This is also true in current `dev`, so this is not a regression.

I guess that the solution would be to encapsulate the `migrate` command under a new node script